app/lib/project/MetapolatorProject.js

```diff
@@ -655,50 +645,56 @@ define([
                                this.fontinfoFileName, this.fontinfoFile );
     };
 
-    function exportInstance(io, project, masterName, dirName, precision) {
-        // returns a models/Controller
-        var model = project.open(masterName)
-          , master = model.query('master#' + masterName)
-          , glyphSet
-          , exportController
-          ;
-
-        // create a bare ufoV2 directory
-        io.mkDir(false, dirName);
-
-        // create dirName/metainfo.plist
-        io.writeFile(false, dirName+'/metainfo.plist'
-                                , plistLib.createPlistString(metainfoV2));
-
-        // fontforge requires a fontinfo.plist that defines unitsPerEm
-        io.writeFile(false, dirName+'/fontinfo.plist'
-                                , plistLib.createPlistString(minimalFontinfo));
+    _p.exportUFOInstance_chunk = function(masterName, dirName, precision) {
```

Comment:
Here is the deal: this method will return `[generator, io]` and the ui will do the rest. Trying to pack all this process state in here is making it more complicated than it must be. Also, it introduces unpleasant side effects, like making `_p.getZippedInstance` unpredictable.

This means also, that you can remove `this.chunk_io` and ` this.chunk_exportController`.

Maybe we call it `getUFOExportGenerator`.
```js
_p.getUFOExportGenerator = function ( masterName, dirName, precision) {
    var io = new InMemory()
      , generator = UFOExportController(io, this, masterName, dirName, precision).exportGenerator()
      ;
    return [generator, io];
}
```